modules/apps/headless/headless-admin-list-type/headless-admin-list-type-test/src/testIntegration/java/com/liferay/headless/admin/list/type/resource/v1_0/test/ListTypeEntryResourceTest.java

```diff
@@ -43,6 +44,32 @@ public void setUp() throws Exception {
 				null, TestPropsValues.getUserId(),
 				Collections.singletonMap(LocaleUtil.getDefault(), "test"),
 				false, Collections.emptyList());
+
+		_systemListTypeDefinition =
+			ListTypeDefinitionLocalServiceUtil.addListTypeDefinition(
+				null, TestPropsValues.getUserId(),
+				Collections.singletonMap(
+					LocaleUtil.getDefault(), RandomTestUtil.randomString()),
+				true, Collections.emptyList());
+	}
+
+	@FeatureFlags("LPD-24055")
+	@Test
+	public void testDeleteSystemListTypeDefinitionListTypeEntry()
+		throws Exception {
+
+		ListTypeEntry listTypeEntry = _addListTypeEntry(
+			_systemListTypeDefinition);
+
+		assertHttpResponseStatusCode(
+			204,
+			listTypeEntryResource.deleteListTypeEntryHttpResponse(
+				listTypeEntry.getId()));
+
+		assertHttpResponseStatusCode(
+			404,
+			listTypeEntryResource.getListTypeEntryHttpResponse(
+				listTypeEntry.getId()));
 	}
```

Comment:
@abesousa For resource tests, we don't create new methods. We reuse the ones from the `BaseResourceTestCase` class.

In this case, can we change it to:

```java
@FeatureFlags("LPD-24055")
@Override
@Test
public void testDeleteListTypeEntry()
	throws Exception {

	super.testDeleteListTypeEntry();

	ListTypeEntry listTypeEntry = _addListTypeEntry(
		_systemListTypeDefinition);

	assertHttpResponseStatusCode(
		204,
		listTypeEntryResource.deleteListTypeEntryHttpResponse(
			listTypeEntry.getId()));

	assertHttpResponseStatusCode(
		404,
		listTypeEntryResource.getListTypeEntryHttpResponse(
			listTypeEntry.getId()));
}
```
?